src/lib/InstantSearch.ts

```diff
@@ -290,20 +290,26 @@ See: https://www.algolia.com/doc/guides/building-search-ui/going-further/backend
   }
 
   /**
-   * Removes a widget. This can be done after the InstantSearch has been started. This feature
-   * is considered **EXPERIMENTAL** and therefore it is possibly buggy, if you find anything please
-   * [open an issue](https://github.com/algolia/instantsearch.js/issues/new?title=Problem%20with%20removeWidget).
-   * @param  {Widget} widget The widget instance to remove from InstantSearch. This widget must implement a `dispose()` method in order to be gracefully removed.
+   * Removes a widget. This can be done after the InstantSearch has been started.
+   *
+   * @param widget The widget instance to remove from InstantSearch.
+   *
+   * This widget must implement a `dispose()` method in order to be gracefully removed.
    */
   public removeWidget(widget: Widget) {
+    warning(
+      false,
+      'removeWidget(widget) is replaced by removeWidgets([widgets]) to help see the structure'
+    );
     this.removeWidgets([widget]);
   }
 
   /**
-   * Removes multiple widgets. This can be done only after the InstantSearch has been started. This feature
-   * is considered **EXPERIMENTAL** and therefore it is possibly buggy, if you find anything please
-   * [open an issue](https://github.com/algolia/instantsearch.js/issues/new?title=Problem%20with%20addWidgets).
-   * @param {Widget[]} widgets Array of widgets instances to remove from InstantSearch.
+   * Removes multiple widgets. This can be done only after the InstantSearch has been started.
```

Comment:
```suggestion
   * Removes multiple widgets from the search instance.
```

We can also mention that it clears the state that the widget is responsible for.